Code/Framework/AzToolsFramework/AzToolsFramework/Slice/SliceDependencyBrowserComponent.cpp

```diff
@@ -224,12 +232,25 @@ namespace AzToolsFramework
     {
         bool found = false;
         bool succeeded = m_databaseConnection->QuerySourceDependencyByDependsOnSource(
-            relativePath.c_str(), "%.slice",
+            relativePath.c_str(),
             AzToolsFramework::AssetDatabase::SourceFileDependencyEntry::DEP_SourceOrJob,
             [&](AssetDatabase::SourceFileDependencyEntry& entry)
         {
+            AzToolsFramework::AssetDatabase::SourceDatabaseEntry sourceEntry;
+            m_databaseConnection->QuerySourceBySourceGuid(entry.m_sourceGuid, [&sourceEntry](AzToolsFramework::AssetDatabase::SourceDatabaseEntry& entry)
+            {
+                sourceEntry = entry;
+                return false;
+            });
+
+            if(!sourceEntry.m_sourceName.ends_with(".slice"))
```

Comment:
.slice files should be replaced by .prefab files at some point, it seems strange to continue to support .slice files

Reply:
Agreed, but since slices aren't totally gone yet I didn't want to just break this tool and if the prefab team wants to convert this over at some point, they'll have something working to start from